### DISCUSSION TITLE
Using the VectorNeumannBC

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#27105] [Q&A Modules: Navier-Stokes] [answered]
I am attempting to use the VectorNeumannBC with the vector velocity. I get the following error:
No standard variable named velocity found. Did you specify a vector variable when you meant to specify a standard variable?

Is it possible to use VectorNeumannBC with the NS variable velocity?

**Comment by GiudGiud** [comment]
Which NS? INSAD or INSFE?

VectorNeumannBC is actually not for vector variables
For NS applications you ll have more specific BCs

**Reply by TLWise** [reply]
@GiudGiud INSAD. I'm looking the implement the Directional Do Nothing BC.

**Reply by maxnezdyur** [reply, marked as answer]
You want to look at `ADVectorFunctionNeumannBC`.  This bit me before too!
